src/realm/object-store/c_api/query.cpp

```diff
@@ -244,11 +244,15 @@ RLM_API realm_query_t* realm_query_parse_for_list(const realm_list_t* list, cons
                                                   const realm_query_arg_t* args)
 {
     return wrap_err([&]() {
+        auto existing_query = list->get_query();
```

Comment:
I guess this is part of another PR

Reply:
oh yes sorry, I have multiple small PRs open... my bad. let me revert this..

Reply:
Code has been reverted